docs/source/workflow_inputs.rst

```diff
@@ -821,6 +822,17 @@ If UFactor and SHGC are not provided, they are defaulted as follows:
 
   OpenStudio-HPXML will return an error if the combination of window properties is not in the above table.
 
+If a storm window is specified, additional information is entered in ``StormWindow``.
+
+  ============================  ========  ======  ===========  ========  =======  ========================================================
+  Element                       Type      Units   Constraints  Required  Default  Notes
+  ============================  ========  ======  ===========  ========  =======  ========================================================
+  ``GlassType``                 string            See [#]_     Yes                Type of storm window glass
+  ============================  ========  ======  ===========  ========  =======  ========================================================
+
+  .. [#] GlassType choices are "clear" or "low-e".
+         The ``UFactor`` and ``SHGC`` of the ``Window`` will be adjusted depending on the ``GlassType``, based on `the linear correlation <https://github.com/NREL/OpenStudio-HEScore/pull/319>`_. 
```

Comment:
The OS-HEScore repo is private, so no one will be able to see the correlation. Please type out the correlation in the documentation and reference the PNNL paper as the source.